room/integration-tests/testapp/src/androidTest/java/androidx/room/integration/testapp/test/DatabaseCallbackTest.java

```diff
@@ -205,6 +211,82 @@ public void onDestructiveMigration_calledOnUpgrade() {
         database.close();
     }
 
+    @Test
+    public void onCreateFromAsset_calledOnOpenPrepackagedDatabase() {
+        Context context = ApplicationProvider.getApplicationContext();
```

Comment:
Might want to do this in an `@After` block so tests can be idempotent.

Reply:
I agree but I wanted to keep new tests consistent with the existing ones so I kept it this way